### ISSUE TITLE
Thumbnail and Title of Result Link to the Same Place

### ISSUE THREAD

**Issue by mtribone** [projectblacklight/blacklight#2037]
When thumbnails are configured to appear with the search index results, both the title and the thumbnail link to the same place. When adjacent links go to the same location this results in additional navigation and repetition for keyboard and screen reader users. It's been a long standing issue in the Samvera community–one I have yet to solve adequately.

https://www.w3.org/TR/WCAG20-TECHS/H2.html

It would be better to combine the thumbnail and the title into the same `<a>` element.

**Comment by jendiamond** [comment]
We are currently having an issue with this in our project. We would like the alt tag to be able to be either "thumbnail" or the title of the image. Mostly we want what will work best for accessibility.

**Comment by cbeer** [comment]
At some point recently, we marked the thumbnail link as `aria-hidden` and took it out of the tab order by default.
